common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -1305,4 +1305,8 @@ object Utils extends Logging {
         (key, value)
     }.asInstanceOf[Seq[(K, V)]]
   }
+
+  def appUniqueIdWithUUID(appUiqueId: String): String = {
+    appUiqueId + "-" + UUID.randomUUID()
```

Comment:
Could you add config to allow user to add UUID suffix? IMO, not all cases should add UUID suffix. Right?

Reply:
better to move appUniqueIdWithUUID(appUiqueId: String) to CelebornConf. Additionally, we can simplify the UUID by removing the dashes to create a shorter version, then we can use celebornConf.appUniqueIdWithUUIDSuffix

```suggestion
  def appUniqueIdWithUUIDSuffix(appId: String): String = {
     if (clientApplicationUUIDSuffixEnabled) {
        appId + "-" + UUID.randomUUID().toString().replaceAll("-", "")
     } else {
       appId
     }
   }

```